client/src/main/java/io/confluent/kafka/schemaregistry/client/MockSchemaRegistryClient.java

```diff
@@ -348,11 +348,18 @@ public Collection<SubjectVersion> getAllVersionsById(int id) throws IOException,
       RestClientException {
     return idToSchemaCache.entrySet().stream()
         .filter(entry -> entry.getValue().containsKey(id))
```

Comment:
@rayokota Can we be sure that the Map returned e.getValue() will not be NULL?

Reply:
Yes, we don't pull null vaules in the `idToSchemaCache` map